draft-ietf-tls-rfc8447bis.md

```diff
@@ -535,6 +535,19 @@ IANA is requested to add a "Comment" column to the following registries:
 This list of registries is all registries that do not already have a
 "Comment" or "Notes" column or that were not orphaned by TLS 1.3.
 
+# Expert Review of Current and Potential IETF and IRTF Documents
+
+The intent of the Specification Required standard for TLS code points
+is to allow for easy registration for code points associated with
+protocols and algorithms that are not being actively developed inside
+IETF or IRTF. When TLS-based technologies are being developed inside
+the IRTF/IETF they should be done in coordination with the TLS WG in
+order to provide appropriate review. For this reason, designated
+experts should discourage code point registrations for documents which
```

Comment:
```suggestion
experts should decline code point registrations for documents which
```

Reply:
I think Sean's revision encourages the behavior that we want.  If an IRTF/IETF document has not be discussed with the TLS working group then it should be declined until that discussion happens.  The result of the discussion might be a go ahead to publish or some action to correct issues with the document.

Reply:
LGTM.